Docs/source/theory/multiphysics/collisions.rst

```diff
@@ -121,13 +121,21 @@ The particle velocity in the COM frame is then isotropically scattered using the
 Back scattering
 ^^^^^^^^^^^^^^^
 
-The process is the same as for elastic scattering above expect the scattering angle is fixed at :math:`\pi`, meaning the particle velocity in the COM frame is updated to :math:`-\vec{u}_c`.
+The process is the same as for elastic scattering above except the scattering angle is fixed at :math:`\pi`, meaning the particle velocity in the COM frame is updated to :math:`-\vec{u}_c`.
 
 Excitation
 ^^^^^^^^^^
 
 The process is also the same as for elastic scattering except the excitation energy cost is subtracted from the particle energy. This is done by reducing the velocity before a scattering angle is chosen.
 
+Forward scattering
```

Comment:
Could you please add a citation to the paper you mentioned (https://doi.org/10.1088/0963-0252/25/5/055026) in this section? I know we don't have citations for the other processes, but seeing as there is one available for this, we might as well add it.

We usually add references in https://github.com/ECP-WarpX/WarpX/blob/development/Docs/source/refs.bib and then use sphinx commands to cite the appropriate reference.